Toolchain/CMake/LLVMConfig.cmake

```diff
@@ -1,5 +1,10 @@
 # This file specifies the options used for building the Clang compiler, LLD linker and the compiler builtins library
 
+if (CMAKE_VERSION VERSION_EQUAL "3.29.0" OR CMAKE_VERSION VERSION_EQUAL "3.29.1")
+    message(FATAL_ERROR "CMake versions 3.29.0 and 3.29.1 are known to not install LLVM correctly. "
+                        "Please either downgrade CMake or update it to 3.29.2+.")
+endif()
```

Comment:
I found that the Serenity build itself had trouble with these cmake versions. we should ban them in either Serenity/CMakeLists.txt or Meta/serenity.sh in a follow-up.